docs/protocol/terminology.mdx

```diff
@@ -0,0 +1,49 @@
+---
+sidebar_position: 2
+title: "Terminology"
+---
+
+import Tooltip from '@site/src/components/Tooltip';
+
+# Terminology
+
+## Core Concepts
+
+**Channel**: A secure communication pathway between participants that locks funds in an on-chain smart contract while enabling off-chain state updates.
+
+**State**: A snapshot of the channel at a point in time, including fund allocations and application-specific data.
+
+**Participant**: An entity (identified by a wallet address) that is part of a channel. Typically includes a Creator (user) and a clearnode or a Clearnode Operator.
```

Comment:
I'd just say Clearnode Operator, as clearnode itself is a logical component, and we usually called this participant as broker, but now we may reconsider.

Reply:
[2407a50f77b866e3cddc84621a441c69e6787515>](https://github.com/layer-3/docs/pull/69/commits/2407a50f77b866e3cddc84621a441c69e6787515)